tasks/jitsi-meet.yml

```diff
@@ -1,15 +1,14 @@
 ---
 
-- name: Fetch signing key for Jitsi repository
-  apt_key:
-    id: "66A9CD0595D6AFA247290D3BEF8B479E2DC1389C"
-    url: "https://download.jitsi.org/jitsi-key.gpg.key"
-    state: present
+- name: Copy Jitsi repository key
+  copy:
+    src: 'files/66A9CD0595D6AFA247290D3BEF8B479E2DC1389C.gpg'
+    dest: '/usr/share/keyrings/66A9CD0595D6AFA247290D3BEF8B479E2DC1389C.gpg'
 
 - name: Add Jitsi repository
-  apt_repository:
-    repo: "deb https://download.jitsi.org/ stable/"
```

Comment:
I would not change that, when we go further with the `apt_key` module. A good possibility would be to make the `repo` string to a variable to make it possible to allow the testing/beta repo from Jitsi.

Reply:
same goes for this, adding the "signed-by" prevents other keys to sign/fake packages from this repository.